hazelcast/src/main/java/com/hazelcast/jet/server/JetCommandLine.java

```diff
@@ -176,7 +176,7 @@ public JetCommandLine(Function<ClientConfig, JetInstance> jetClientFn, PrintStre
 
     public static void main(String[] args) {
         runCommandLine(
-                config -> HazelcastClient.newHazelcastClient(config).getJetInstance(),
+                config -> (JetInstance) HazelcastClient.newHazelcastClient(config).getJet(),
```

Comment:
The `runCommandLine` could be changed to accept `JetService`

Reply:
I am thinking of converting this class to `HazelcastCommandLine` class. I asked it in slack. If it is decided, I will do it in a separate PR.